cupy/core/raw.pyx

```diff
@@ -83,12 +80,20 @@ cdef class RawKernel:
 
     @property
     def kernel(self):
+        return self._kernel()
+
+    def _kernel(self, options=None, backend=None, log_stream=None):
```

Comment:
I think it's OK to cdef this:
```suggestion
    cdef Function _kernel(self, options=None, backend=None, log_stream=None):
```
(btw, Cython doc says `cdef inline` won't be faster for class cdef methods, so it's ok to not inline.)